<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QbsProjectManager::Internal::QbsBuildStepConfigWidget</class>
 <widget class="QWidget" name="QbsProjectManager::Internal::QbsBuildStepConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>97</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>16</number>
   </property>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="dryRunCheckBox">
        <property name="text">
         <string>Dry run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="keepGoingCheckBox">
        <property name="text">
         <string>Keep Going</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="jobSpinBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>jobs</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="buildVariantComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Debug</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Release</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="buildVariantLabel">
     <property name="text">
      <string>Build variant:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
